source/adapters/opencl/command_buffer.cpp

```diff
@@ -1,71 +1,127 @@
 //===--------- command_buffer.cpp - OpenCL Adapter ---------------------===//
 //
-// Copyright (C) 2023 Intel Corporation
-//
-// Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM
-// Exceptions. See LICENSE.TXT
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
-//===----------------------------------------------------------------------===//
+//===-----------------------------------------------------------------===//
```

Comment:
Why are we touching these license lines? If this is relevant think it should be a standalone PR that updates all the affected files

Reply:
Yeah, this license change should be reverted. The removed license comments are the correct ones for UR.

Reply:
Fixed thanks think this was an accidental copy paste error.